sql/postgres/migrate.go

```diff
@@ -797,6 +802,11 @@ func (s *state) addIndexes(t *schema.Table, indexes ...*schema.Index) {
 			Comment: fmt.Sprintf("create index %q to table: %q", idx.Name, t.Name),
 			Reverse: func() string {
 				b := s.Build("DROP INDEX")
+				if c := (Concurrently{}); sqlx.Has(idx.Attrs, &c) {
+					if c.Concurrently {
+						b.P("CONCURRENTLY")
+					}
+				}
```

Comment:
```suggestion
				if c := (Concurrently{}); sqlx.Has(idx.Attrs, &c) {
					b.P("CONCURRENTLY")
				}
```
See the comment above.